clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -267,6 +272,12 @@
                                         atLeast(0L),
                                         Importance.LOW,
                                         CommonClientConfigs.RECONNECT_BACKOFF_MS_DOC)
+                                .define(RECONNECT_BACKOFF_MAX_CONFIG,
+                                        Type.LONG,
+                                        50L,
```

Comment:
Yes. 50ms is disabling exponential backoff by default. I'm happy to enable it with reasonable defaults. Remember that the actual reconnect is randomized, so for an average of 1sec per reconnect attempt during failure, default should be 2000L.